modules/oadp-ceph-split.adoc

```diff
@@ -0,0 +1,165 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-ceph-preparing-crs_{context}"]
```

Comment:
Please can you check this ID as it looks like it is duplicated in oadp-ceph-preparing-crs.adoc